targets/csharp/source/PlayFabSDK/source/Uunit/tests/PlayFabQosApiTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace PlayFab.UUnit
```

Comment:
This file needs to be wrapped in:
#if !DISABLE_PLAYFABCLIENT_API && !DISABLE_PLAYFABENTITY_API
#endif